packages/jest-preset/jest-preset.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
       'jest-preset-hops/mocks/file.js',
     '^.+\\.tpl$': 'jest-preset-hops/mocks/tpl.js',
     '^.+\\.css$': 'identity-obj-proxy',
+    hops: 'hops/runtime.js',
```

Comment:
Should we do the same for `hops-react` (and if needed other packages as well)?
We should at least remove[ the deep import](https://github.com/xing/hops/blob/2cd4b1162b375a35e4cc964f41d14438db2e3496/packages/react/index.js#L7) again.

Reply:
I would argue against removing the deep import, because `@untool/react`'s `"main"` export is implicitly the [`index.js`](https://github.com/untool/untool/blob/master/packages/react/index.js) and the deep import to `@untool/react/lib/runtime.js` will only be picked up through the `"browser"` and `"server"` fields.
Therefore we won't be able to use the runtime components in jest, because jest does not understand `browser` or `server` (or at least not without configuration - which would complicate other things).

And the `hops-react` package only has a single `"main"` field, wich already contains the runtime components, therefore a jest alias for `hops-react` is not necessary.